backside/README.md

```diff
@@ -0,0 +1,7 @@
+Setup:
+
+    sudo npm i
```

Comment:
```suggestion
    npm i
```

There's no need to use `sudo` with `npm install`. If you get access error while installing, you need to fix it on your system: https://stackoverflow.com/a/16151707